_posts/2019-09-18-uuid4-tiny.md

```diff
@@ -0,0 +1,48 @@
+---
+author: Daniel Mita
+layout: post
+title: UUID4::Tiny
+subtitle: Cryptographically secure version 4 UUIDs via the Linux getrandom() system call
+tags: perl uuid cryptography cpan module
+---
+
+[UUID::Tiny][uuid-tiny] is nice, but unfortunately due to its usage of
+rand() the version 4 UUIDs it provides are not cryptographically
+strong. We opted to create a lightweight (core modules only!)
```

Comment:
we can not promise that the next uuid we generate won't be guessable might be a clearer way to explain it?